frontend/src/components/Common/ReportForThreeColumns/index.tsx

```diff
@@ -13,11 +16,27 @@ interface ReportForThreeColumnsProps {
 }
 
 export const ReportForThreeColumns = ({ title, fieldName, listName, data }: ReportForThreeColumnsProps) => {
+  const emojiRow = (row: ReportDataWithThreeColumns) => {
+    if (row.name.includes(':') && getEmojiUrls(row.name, emojis).length > 0) {
+      const prefix = row.name.split('/')[0]
```

Comment:
Personally I would prefer the destructuring approach here
`const [prefix, suffix] = row.name.split('/')`
as using index to get the array values may slightly reduce readability.

Reply:
nice syntactic sugar!